crypto/mem.c

```diff
@@ -165,13 +166,23 @@ void ossl_malloc_setup_failures(void)
 {
     const char *cp = getenv("OPENSSL_MALLOC_FAILURES");
 
-    if (cp != NULL && (md_failstring = strdup(cp)) != NULL)
+    if (cp != NULL) {
+        md_fs_orig = strdup(cp);
```

Comment:
Please use `OPENSSL_strdup()` - not all systems have `strdup()`

Reply:
sure, easily done (in push below)

Reply:
> Please use `OPENSSL_strdup()` - not all systems have `strdup()`

We cannot use OPENSSL_strdup() here.